### PR TITLE
Add tipping icon

### PR DESCRIPTION
**Pull request by adityatr-ls**
## Description

Which problem(s) does it solve and how? What does this PR add?

Adds Tipping icon for retail
image

## How to test?

- Checkout branch, run `yarn dev`
- Open [Storybook](http://localhost:6006)
- Or check the deploy preview on Netlify (link available in comments)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md) guide
- [x] I have prepared [CHANGELOGs](https://github.com/lightspeed/flame/blob/master/.github/CONTRIBUTING.md#git-workflow) for release
- [x] I have tested my changes on [supported browsers](https://browserl.ist/?q=%3E0.25%25%2C+not+op_mini+all%2C+not+ie+%3C%3D+11)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/flame/svg/Icons/icon-tipping.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><g id="tipping"><path id="base-1" d="M5 9H4l-3 2 2 4h1l2-1 2 1h1l6-3 1-1-1-1h-5l1 2-1 1-5-1v-1l5 1-1-1-4-2z"/><path id="base-2" d="M11 4a2 2 0 10-1 5 2 2 0 001-5zM6 1a2 2 0 100 4 2 2 0 000-4z" opacity=".7" fill-rule="evenodd" clip-rule="evenodd"/></g></svg>
```
Comment (ls-frederic-bouchard) [inline packages/flame/svg/Icons/icon-tipping.svg:1]: - We should not use any `opacity=".7"` here. Instead, find the closest gray/blue color. 
- Is it possible that it's also missing the `fill` attribute on the first path?

**Review by ls-frederic-bouchard — COMMENTED** [review]
Few questions about the SVG. @xdrdak do we have a clear guide on how to set it up. I remember doing some specific changes to it while working with Alexis on that.

File: packages/flame/svg/Icons/icon-tipping.svg (hunk 1 above)
Comment (adityatr-ls) [reply packages/flame/svg/Icons/icon-tipping.svg:1]: @ls-stephan-ye I think can answer these questions better

Comment (ls-frederic-bouchard) [reply packages/flame/svg/Icons/icon-tipping.svg:1]: I'm just saying these are Flame requirements. This has nothing to do with the icon export. **You** have to modify those properties inside the `SVG` code to respects those requirements. Like removing any path with opacity attribute to make sure a solid color is used.

Comment (ls-frederic-bouchard) [reply packages/flame/svg/Icons/icon-tipping.svg:1]: Not respecting these requirements might have an impact on the icon API not returning back the expected results when changing the color palette for example.

Comment (ls-frederic-bouchard) [reply packages/flame/svg/Icons/icon-tipping.svg:1]: Ok, apparently most of the icons use the base `opacity=".7"` on the level `detail-2`. Let's keep it this way then. It's just the background on which the icon is shown, might impact the actual color of the coins in this case.

**Review by xdrdak — APPROVED** [review]

**Pull request merged**